upgrade_tests/upgrade_compact_storage.py

```diff
@@ -178,3 +178,50 @@ def test_upgrade_with_dropped_compact_storage_index(self):
                      [{'col1': '50', 'column1': None, 'pk': '5', 'value': None}])
         assert (list(session.execute("SELECT * FROM ks.compact_table WHERE pk = '5'")) ==
                      [{'col1': '50', 'column1': None, 'pk': '5', 'value': None}])
+
+    def test_downgrade_after_failed_upgrade(self):
```

Comment:
Nice test. Could we add a brief comment mentioning the purpose of the test, especially the part about not spoiling the commit log before checking for `COMPACT STORAGE`, and referencing CASSANDRA-16063?